ftw/tabbedview/browser/tabbed.pt

```diff
@@ -43,7 +43,9 @@
           <div tal:replace="structure provider:plone.abovecontenttitle" />
 
 
-          <metal:title define-slot="content-title">
+          <metal:title define-slot="tabbedview-content-title">
+            <metal:comment>
+            </metal:comment>
```

Comment:
@Rotonen are you going to add a comment here? Or am I too early to look at the PR? 😜